os.bpmsuite.businesscentral/added/launch/bpmsuite-businesscentral.sh

```diff
@@ -15,31 +16,41 @@ function prepareEnv() {
     unset APPFORMER_JMS_BROKER_PASSWORD
     unset APPFORMER_JMS_BROKER_PORT
     unset APPFORMER_JMS_BROKER_USER
+    unset_KIE_SECURITY_ENV
```

Comment:
typo?

Reply:
Not a typo. It's a function in bpmsuite-security.sh.